stl/inc/chrono

```diff
@@ -612,6 +612,11 @@ namespace chrono {
             const long long _Freq = _Query_perf_frequency(); // doesn't change after system boot
             const long long _Ctr  = _Query_perf_counter();
             static_assert(period::num == 1, "This assumes period::num == 1.");
+            // Below algorithm instead of just having "(_Ctr * period::den) / _Freq"
+            // prevents overflow when _Ctr is sufficiently large.
+            // It assumes that _Freq * period::den does not overflow, which is currently true for nano period
+            // It is not realistic for _Ctr to accumulate to large values from zero with this assumption,
+            // but initial _Ctr could be large value.

```

Comment:
I suggest some grammar tweaks. I think this is a great explanation for otherwise-mysterious code, thank you.

```suggestion
            // Instead of just having "(_Ctr * period::den) / _Freq",
            // the algorithm below prevents overflow when _Ctr is sufficiently large.
            // It assumes that _Freq * period::den does not overflow, which is currently true for nano period.
            // It is not realistic for _Ctr to accumulate to large values from zero with this assumption,
            // but the initial value of _Ctr could be large.
```